egs/heroico/s5/local/chain/tuning/run_tdnn_1i.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
+# 1i
+# set number of epochs to 7 (avoid overfitting like 1e)
+# set network hidden layer dimensions to 256
+
+# info
+# exp/chain/tdnn1i_sp:
+# num-iters=105
+# nj=1..1
+# num-params=2.0M
+# dim=40+100->1392
```

Comment:
Since this is a new recipe, I'd prefer to have just a single recipe here, whichever is the best, and call it 1a.